[BUG] Sklearn update breaks our unit tests

## Expected Behavior
`make tests` should complete successfully with latest sklearn (0.24.0+)

## Actual Behavior
```
tests/test_sklearn.py:9: in <module>
    check_estimator(graspologic.embed.LaplacianSpectralEmbed)
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/sklearn/utils/estimator_checks.py:532: in check_estimator
    raise TypeError(msg)
E   TypeError: Passing a class was deprecated in version 0.23 and isn't supported anymore from 0.24.Please pass an instance instead.
```

## Your Environment
* Python version: 3.6, 3.7, 3.8
* graspologic version: latest

While working on #611 the unit tests started failing.  I limited us to an sklearn version under 0.24.0 for that PR but we should address this and fix the upward bound in setup.py

--- a/tests/test_mug2vec.py
+++ b/tests/test_mug2vec.py
@@ -13,8 +13,8 @@ from graspologic.simulations import sbm
 def generate_data():
     np.random.seed(1)
 
-    p1 = [[0.2, 0.1], [0.1, 0.2]]
-    p2 = [[0.1, 0.2], [0.2, 0.1]]
+    p1 = [[0.3, 0.1], [0.1, 0.3]]
+    p2 = [[0.1, 0.3], [0.3, 0.1]]
     n = [50, 50]
 
     g1 = [sbm(n, p1) for _ in range(20)]

--- a/tests/test_sklearn.py
+++ b/tests/test_sklearn.py
@@ -6,5 +6,5 @@ import numpy as np
 
 from sklearn.utils.estimator_checks import check_estimator
 
-check_estimator(graspologic.embed.LaplacianSpectralEmbed)
-check_estimator(graspologic.embed.ClassicalMDS)
+check_estimator(graspologic.embed.LaplacianSpectralEmbed())
+check_estimator(graspologic.embed.ClassicalMDS())

